lib/model/store.dart

```diff
@@ -319,6 +320,8 @@ class PerAccountStore extends ChangeNotifier with StreamStore, MessageStore {
 
   final Map<int, User> users;
 
+  final RecentSenders recentSenders = RecentSenders();
```

Comment:
It's true that this can be seen as "data about" users, as the section heading here says — but it's also "summaries of messages". Let's put it in that latter section, because the things it most resembles are Unreads and especially RecentConversationsView.